Add financial report (#69)
rust-analyzer/rust-analyzer.github.io: blog/_posts/2020-08-29-financial-report.adoc

= `2020[..6]` Financial Report
:sectanchors:
:page-layout: post

This is a short financial report for https://opencollective.com/rust-analyzer/[Open Collective] for the first half of 2020, from 01-01-2020 until 30-06-2020.

As we haven't noted the exact date of the previous report, the numbers in this one will be approximate :(
It probably is good that @matklad didn't choose accounting as a career.
The next report should be exact though.

We've got about 14000  EUR on open collective after fees.

> Of which, we distributed 12000 EUR by splitting it into two buckets:

Reviewer: ```suggestion
Of which we distributed 12000 EUR by splitting it into two buckets:
```

* 50% for Ferrous Systems
* 50% for contributors, not affiliated with Ferrous

We were able to pay more than our usual 20% to contributors because we managed to do some direct contracting work for rust-analyzer.

Ferrous share went to paying for part of https://github.com/matklad[@matklad]'s time and for administrative tasks.

> 3000 EUR were payed to https://github.com/flodiebold[@flodiebold] for chalk integration and various other type system improvements.

Reviewer: ```suggestion
3000 EUR were paid to https://github.com/flodiebold[@flodiebold] for chalk integration and various other type system improvements.
```

> 3000 EUR were payed to https://github.com/edwin0cheng[@edwin0cheng] for implementing most of declarative and procedural macro support.

Reviewer: ```suggestion
3000 EUR were paid to https://github.com/edwin0cheng[@edwin0cheng] for implementing most of declarative and procedural macro support.
```

Our biggest Open Collective sponsors in the first half of 2020 are:

* https://www.embark-studios.com[Embark Studios] (6000EUR)
* https://discordapp.com[Discord] (1500EUR)
* https://parity.io[Parity] (1500EUR)
* https://www.prisma.io/[Prisma] (1200EUR)
* https://github.com/repi[Johan Andersson of Embark] (1100EUR)

<3

If you use rust-analyzer professionally, consider asking your company to https://opencollective.com/rust-analyzer/[sponsor] the development of this project.

> We are also planing to announce a couple of new ways to support rust-analyzer development soon, stay tuned!

Reviewer: ```suggestion
We are also planning to announce a couple of new ways to support rust-analyzer development soon, stay tuned!
```

> As a reminder, we publish https://rust-analyzer.github.io/thisweek[weekly changelog] to report the progress on rust-analyzer and explain where the money go.

Reviewer: ```suggestion
As a reminder, we publish https://rust-analyzer.github.io/thisweek[weekly changelogs] to report the progress on rust-analyzer and explain where the money goes.
```
